simple-marketplace/hooks/useMetadatabyMetadataId.ts

```diff
@@ -0,0 +1,54 @@
+/*
+
+useMetadataByMetadataId Hook
+Description: This hook calls metadataByMetadataById method from @mintbase-js/data to get the information of the current NFT opened on the BuyModal.
+
+*/
+
+import { metadataByMetadataId } from '@mintbase-js/data';
+import { useQuery } from 'react-query';
+import { parseYactoToNear } from '../lib/numbers';
+import { SelectedNft, TokenListData } from '../types/types';
+
+const useMetadataByMetadataId = ({
+  metadataId,
+}: SelectedNft): Partial<TokenListData> => {
+  const {
+    isLoading,
+    data: metadata,
+  } = useQuery('metadataByMetadataId', () => metadataByMetadataId(metadataId), {
+    retry: false,
+    staleTime: Infinity,
```

Comment:
```
export const staleTime = {
    Default: 60 * 1000,
    Infinite: Infinity,
};
```
I wouldnt add Infinity, as the user might want to the data be refetched after like .. 1min